src/main/resources/templates/cluster-overview.ftl

```diff
@@ -89,7 +89,7 @@
             <#list brokers as b>
                 <tr>
                     <td><a href="<@spring.url '/broker/${b.id}'/>"><i class="fa fa-info-circle fa-lg"></i> ${b.id}</a></td>
-                    <td>${b.host}</td>
+                    <td>${b.host?if_exists}</td>
```

Comment:
Is there a legitimate reason why `host` would be `null` at this point?

Reply:
since Kafka-utils is not officially compatible with Kafka >= 0.11 yet.
i am using kafka 2.3.0
the similiar issue had also created on the following, detail can refer it.
https://github.com/Yelp/kafka-utils/issues/157